lib/router/website_router_delegate.dart

```diff
@@ -0,0 +1,120 @@
+import 'package:festival_flutterturkiye_org/core/model/speaker.dart';
+import 'package:festival_flutterturkiye_org/core/ui/speaker_detail_content.dart';
+import 'package:festival_flutterturkiye_org/pages/home_page.dart';
+import 'package:festival_flutterturkiye_org/pages/unknown_page.dart';
+import 'package:festival_flutterturkiye_org/router/website_configuration.dart';
+import 'package:flutter/material.dart';
+
+class WebsiteRouterDelegate extends RouterDelegate<WebsiteConfiguration>
+    with ChangeNotifier, PopNavigatorRouterDelegateMixin<WebsiteConfiguration> {
+  WebsiteRouterDelegate() {
+    _homePage = MaterialPage(
+      key: const ValueKey<String>('HomePage'),
+      child: HomePage(
+        pageTitleNotifier: _pageTitleNotifier,
```

Comment:
aren't these fields at this time null as the constructor is created but the field assignments are afterwards? But not sure right now 🤔

Reply:
I think the order of the declaration matters